src/router-hooks.tsx

```diff
@@ -95,28 +105,28 @@ export const useRouterState = (): RouterContextValue => {
 };
 
 export const toUrl = (state: RouterState): string => {
-  const query = Object.entries(state)
-    .filter(([k, v]) => k !== "focus" && !!v)
-    .map(([k, v]) => {
-      return `${encodeURIComponent(k)}=${encodeURIComponent(
-        serializeValue(v)
-      )}`;
-    })
-    .join("&");
-  return window.location.toString().split("?")[0] + (query ? "?" + query : "");
+  // This could be cleaned up if we always set the tab.
+  const parts = [
+    state.tab ??
+      (state.tab === "api" || state.api ? "api" : undefined) ??
```

Comment:
```suggestion
      (state.api ? "api" : undefined) ??
```

We already addressed the tab-is-set case.